flang/tools/bbc/bbc.cpp

```diff
@@ -371,7 +371,7 @@ static mlir::LogicalResult convertFortranSourceToMLIR(
       ctx, semanticsContext, defKinds, semanticsContext.intrinsics(),
       semanticsContext.targetCharacteristics(), parsing.allCooked(),
       targetTriple, kindMap, loweringOptions, envDefaults,
-      semanticsContext.languageFeatures(), targetMachine);
+      semanticsContext.languageFeatures(), targetMachine, "");
```

Comment:
NIT: Would it be worth creating a `constexpr const char* tuneCPU = ""` and using that in place of the empty string literal here? It might make it clear what that parameter is. 

I don't have a strong opinion on this one way or another. If you or anyone else has a different opinion, I'm happy to defer to it.

Reply:
Thank you Tarun, I think that's a much better way to do this. I have made the change.